src/main/java/io/confluent/connect/jdbc/sink/DbStructure.java

```diff
@@ -172,6 +172,33 @@ boolean amendIfNecessary(
         missingFields.add(field);
       }
     }
-    return missingFields;
+
+    // check if the missing fields can be located by ignoring case
+    Set<String> columnNamesLowerCase = new HashSet<>();
+    for (String columnName: dbColumnNames) {
+      columnNamesLowerCase.add(columnName.toLowerCase());
+    }
+
+    if (columnNamesLowerCase.size() != dbColumnNames.size()) {
+      log.warn("Table has column names which differ only with case sensitivity. Original "
+          + "columns={}", dbColumnNames);
```

Comment:
Perhaps different wording:

    Table has column names that differ from field names only by case. Original ...